server-sdk/checkstyle.xml

```diff
@@ -0,0 +1,267 @@
+<?xml version="1.0"?>
```

Comment:
Why copy pmd/checkstyle config file in this folder? Can you reuse existing one?

Reply:
I wanted to control the configruations of the style. Things like number of chars in a line etc. So I have this

Reply:
We format the code using Google style, that's why we disable all those style checking.
Google philosophy is force the whole world using single style and not allow customization.

Reply:
True, but configuring the checkstyle becomes easier with the file.